README.md

```diff
@@ -1,105 +1,128 @@
-
-We understand that users concerned with their privacy prefer to keep their code on their own machine. Rest assured that we’re taking the following steps to address this concern:
-For individual developers, we are working on a reduced-size model which can run on a laptop with reasonable latency. Update: we’ve released Tabnine Local.
-For enterprise users, we will soon roll-out the option to license the model and run it on your own hardware. We can also train a custom model for you which understands the unique patterns and style within your codebase. If this could be relevant to you or your team, we would love to hear more about your use case at enterprise@tabnine.com.
-Enabling Tabnine Deep Cloud sends small parts of your code to our servers to provide GPU-accelerated completions.
-Other than for the purpose of fulfilling your query, your data isn’t used, saved or logged in any way.
-
-**[You can enable Tabnine Deep Cloud for free by signing up Tabnine Professional](https://www.tabnine.com/trial/)**
-
-**[You can read more about Deep Completion](https://www.tabnine.com/blog/deep)**
-
-### **Frequently Asked Questions:**
-
----
-
-<details><summary>Is Tabnine simple to use?</summary>
-<p>Tabnine works for all programming languages.
-Tabnine does not require any configuration in order to work.
-Tabnine does not require any external software (though it can integrate with it).
-Since Tabnine does not parse your code, it will never stop working because of a mismatched bracket.
-</p>
-</details>
-
-<details><summary>Will Tabnine leak my source code?</summary>
-<p>By default, Tabnine makes web requests only for the purposes of downloading updates and validating registration keys. In this case your code is not sent anywhere, even to Tabnine servers.
-You may opt in to Tabnine Deep Cloud, which allows you to use TabNine’s servers for GPU-accelerated completions powered by a deep learning model. If sending code to a cloud service is not possible, we also offer a self-hosted option. 
-Contact us at enterprise@tabnine.com.
-</p>
-</details>
-
-### **Note:**
-
----
-
-A note on licensing: this repo includes source code as well as packaged Tabnine binaries. The MIT license only applies to the source code, not the binaries. The binaries are covered by the [Tabnine End User License Agreement](https://tabnine.com/eula).
+​
+**Deep Completion**
+​
+Deep Tabnine is trained on millions of files from GitHub. During training, Tabnine’s goal is to predict the next token given the tokens that came before. To achieve this goal, Tabnine learns complex behavior, such as type inference in dynamically typed languages.
+​
+Deep Tabnine can use subtle clues that are difficult for traditional tools to access. For example,
+the return type of app.get_user()is assumed to be an object with setter methods, while the return type of app.get_users() is assumed to be a list.
```

Comment:
Wrap code with backticks eg: 
return type of `app.get_user()` is assumed to be an object with setter methods, while the return type of `app.get_users()` is assumed to be a list.